--- origsrc/git-1.7.9/config.mak.uname	2012-01-27 13:51:04.000000000 -0700
+++ src/git-1.7.9/config.mak.uname	2012-02-11 11:56:20.765625000 -0700
@@ -989,12 +989,10 @@ ifeq ($(uname_O),Cygwin)
 	endif
 	NEEDS_LIBICONV = YesPlease
 	NO_FAST_WORKING_DIRECTORY = UnfortunatelyYes
-	NO_TRUSTABLE_FILEMODE = UnfortunatelyYes
 	NO_ST_BLOCKS_IN_STRUCT_STAT = YesPlease
 	# There are conflicting reports about this.
 	# On some boxes NO_MMAP is needed, and not so elsewhere.
 	# Try commenting this out if you suspect MMAP is more efficient
-	NO_MMAP = YesPlease
 	X = .exe
 	COMPAT_OBJS += compat/cygwin.o
 	UNRELIABLE_FSTAT = UnfortunatelyYes
--- origsrc/git-1.7.9/git-gui/Makefile	2012-01-27 13:51:04.000000000 -0700
+++ src/git-1.7.9/git-gui/Makefile	2012-02-11 12:07:50.890625000 -0700
@@ -136,7 +136,7 @@ GITGUI_SCRIPT   := $$0
 GITGUI_RELATIVE :=
 GITGUI_MACOSXAPP :=
 
-ifeq ($(uname_O),Cygwin)
+ifeq ($(uname_O),noThanks)
 	GITGUI_SCRIPT := `cygpath --windows --absolute "$(GITGUI_SCRIPT)"`
 
 	# Is this a Cygwin Tcl/Tk binary?  If so it knows how to do
