--- a/git-v2.51.0/t/t3070-wildmatch.sh	2022-12-11 00:32:48.000000000 +0000
+++ b/git-v2.51.0/t/t3070-wildmatch.sh	2022-12-11 15:08:33.604989200 +0000
@@ -119,7 +119,18 @@
 	then
 		if test -e .git/created_test_file
 		then
-			test_expect_success EXPENSIVE_ON_WINDOWS "$match_function (via ls-files): match '$pattern' '$text'" "
+			# https://github.com/me-and/Cygwin-Git/issues/42
+			# This is a very rough first attempt to characterise
+			# what will and will not cause faulures; it may be
+			# completely incorrect.
+			if [[ "$pattern" = *\\* ]] && [[ "$pattern" != *\\\\-* ]] && [[ "$pattern" != *\\\\,* ]] && [[ "$pattern" != *\[\\\[* ]] && ! [[ "$pattern" = '[\1-\3]' && "$text" = 3 ]]
+			then
+				local result=failure
+			else
+				local result=success
+			fi
+
+			test_expect_$result EXPENSIVE_ON_WINDOWS "ALPHA $match_function (via ls-files): match '$pattern' '$text'" "
 				printf '%s' '$text' >expect &&
 				git$ls_files_args ls-files -z -- '$pattern' >actual.raw 2>actual.err &&
 				$match_stdout_stderr_cmp
@@ -131,7 +142,17 @@
 	then
 		if test -e .git/created_test_file
 		then
-			test_expect_success EXPENSIVE_ON_WINDOWS "$match_function (via ls-files): no match '$pattern' '$text'" "
+			# https://github.com/me-and/Cygwin-Git/issues/42
+			# This is a very rough first attempt to characterise
+			# what will and will not cause faulures; it may be
+			# completely incorrect.
+			if [[ "$pattern" = *\\\\* ]]
+			then
+				local result=failure
+			else
+				local result=success
+			fi
+			test_expect_$result EXPENSIVE_ON_WINDOWS "$match_function (via ls-files): no match '$pattern' '$text'" "
 				>expect &&
 				git$ls_files_args ls-files -z -- '$pattern' >actual.raw 2>actual.err &&
 				$match_stdout_stderr_cmp
