--- origsrc/git-v2.51.0/t/t6137-pathspec-wildcards-literal.sh	2025-10-17 12:20:14.288838800 +0100
+++ src/git-v2.51.0/t/t6137-pathspec-wildcards-literal.sh	2025-10-17 12:17:41.278986400 +0100
@@ -39,7 +39,8 @@ test_expect_success 'add wildcard *' '
 	)
 '
 
-test_expect_success 'add literal \*' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'add literal \*' '
 	git init test-asterisk-literal &&
 	(
 		cd test-asterisk-literal &&
@@ -125,7 +126,8 @@ test_expect_success 'add wildcard f*' '
 	)
 '
 
-test_expect_success 'add literal f\*' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'add literal f\*' '
 	git init test-f-lit &&
 	(
 		cd test-f-lit &&
@@ -156,7 +158,8 @@ test_expect_success 'add wildcard f**' '
 	)
 '
 
-test_expect_success 'add literal f\*\*' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'add literal f\*\*' '
 	git init test-fdstar-lit &&
 	(
 		cd test-fdstar-lit &&
@@ -184,7 +187,8 @@ test_expect_success 'add wildcard f?z' '
 	)
 '
 
-test_expect_success 'add literal \? literal' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'add literal \? literal' '
 	git init test-q-lit &&
 	(
 		cd test-q-lit &&
@@ -227,7 +231,8 @@ test_expect_success 'add wildcard hello?
 	)
 '
 
-test_expect_success 'add literal hello\?world' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'add literal hello\?world' '
 	git init test-hellolit &&
 	(
 		cd test-hellolit &&
@@ -241,7 +246,8 @@ test_expect_success 'add literal hello\?
 	)
 '
 
-test_expect_success 'add literal [abc]' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'add literal [abc]' '
 	git init test-brackets-lit &&
 	(
 		cd test-brackets-lit &&
@@ -280,7 +286,8 @@ test_expect_success 'commit: wildcard *'
 	)
 '
 
-test_expect_success 'commit: literal *' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'commit: literal *' '
 	git init test-c-asterisk-lit &&
 	(
 		cd test-c-asterisk-lit &&
@@ -313,7 +320,8 @@ test_expect_success 'commit: wildcard f*
 	)
 '
 
-test_expect_success 'commit: literal f\*' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'commit: literal f\*' '
 	git init test-c-flit &&
 	(
 		cd test-c-flit &&
@@ -346,7 +354,8 @@ test_expect_success 'commit: wildcard pa
 	)
 '
 
-test_expect_success 'commit: literal f\*\*' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'commit: literal f\*\*' '
 	git init test-c-fdstar-lit &&
 	(
 		cd test-c-fdstar-lit &&
@@ -379,7 +388,8 @@ test_expect_success 'commit: wildcard ?'
 	)
 '
 
-test_expect_success 'commit: literal \?' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'commit: literal \?' '
 	git init test-c-qlit &&
 	(
 		cd test-c-qlit &&
@@ -411,7 +421,8 @@ test_expect_success 'commit: wildcard he
 	)
 '
 
-test_expect_success 'commit: literal hello\?world' '
+# https://github.com/cygporter/git/issues/63
+test_expect_failure 'commit: literal hello\?world' '
 	git init test-c-hellolit &&
 	(
 		cd test-c-hellolit &&
