--- a/git-v2.51.0/t/t0301-credential-cache.sh	2022-12-11 00:32:48.000000000 +0000
+++ b/git-v2.51.0/t/t0301-credential-cache.sh	2022-12-11 13:31:52.820186800 +0000
@@ -42,7 +42,8 @@
 helper_test_oauth_refresh_token cache
 helper_test_authtype cache
 
-test_expect_success 'socket defaults to ~/.cache/git/credential/socket' '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure 'socket defaults to ~/.cache/git/credential/socket' '
 	test_when_finished "
 		git credential-cache exit &&
 		rmdir -p .cache/git/credential/
@@ -56,7 +57,8 @@
 # test behavior when XDG_CACHE_HOME is set
 helper_test cache
 
-test_expect_success "use custom XDG_CACHE_HOME if set and default sockets are not created" '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure "use custom XDG_CACHE_HOME if set and default sockets are not created" '
 	test_when_finished "git credential-cache exit" &&
 	test_path_is_socket "$XDG_CACHE_HOME/git/credential/socket" &&
 	test_path_is_missing "$HOME/.git-credential-cache/socket" &&
@@ -64,7 +66,8 @@
 '
 unset XDG_CACHE_HOME
 
-test_expect_success 'credential-cache --socket option overrides default location' '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure 'credential-cache --socket option overrides default location' '
 	test_when_finished "
 		git credential-cache exit --socket \"\$HOME/dir/socket\" &&
 		rmdir \"\$HOME/dir\"
@@ -78,7 +81,8 @@
 	test_path_is_socket "$HOME/dir/socket"
 '
 
-test_expect_success "use custom XDG_CACHE_HOME even if xdg socket exists" '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure "use custom XDG_CACHE_HOME even if xdg socket exists" '
 	test_when_finished "
 		git credential-cache exit &&
 		sane_unset XDG_CACHE_HOME
@@ -101,7 +105,8 @@
 	test_path_is_socket "$XDG_CACHE_HOME/git/credential/socket"
 '
 
-test_expect_success 'use user socket if user directory exists' '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure 'use user socket if user directory exists' '
 	test_when_finished "
 		git credential-cache exit &&
 		rmdir \"\$HOME/.git-credential-cache/\"
@@ -117,7 +122,8 @@
 	test_path_is_socket "$HOME/.git-credential-cache/socket"
 '
 
-test_expect_success SYMLINKS 'use user socket if user directory is a symlink to a directory' '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure SYMLINKS 'use user socket if user directory is a symlink to a directory' '
 	test_when_finished "
 		git credential-cache exit &&
 		rmdir \"\$HOME/dir/\" &&

