--- a/git-v2.51.0/t/t0301-credential-cache.sh	2022-12-11 00:32:48.000000000 +0000
+++ b/git-v2.51.0/t/t0301-credential-cache.sh	2022-12-11 13:31:52.820186800 +0000
@@ -42,7 +42,8 @@
 helper_test_oauth_refresh_token cache
 helper_test_authtype cache
 
-test_expect_success 'socket defaults to ~/.cache/git/credential/socket' '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure 'socket defaults to ~/.cache/git/credential/socket' '
 	test_when_finished "
 		git credential-cache exit &&
 		rmdir -p .cache/git/credential/
@@ -56,7 +57,8 @@
 # test behavior when XDG_CACHE_HOME is set
 helper_test cache
 
-test_expect_success "use custom XDG_CACHE_HOME if set and default sockets are not created" '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure "use custom XDG_CACHE_HOME if set and default sockets are not created" '
 	test_when_finished "git credential-cache exit" &&
 	test_path_is_socket "$XDG_CACHE_HOME/git/credential/socket" &&
 	test_path_is_missing "$HOME/.git-credential-cache/socket" &&
@@ -64,7 +66,8 @@
 '
 unset XDG_CACHE_HOME
 
-test_expect_success 'credential-cache --socket option overrides default location' '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure 'credential-cache --socket option overrides default location' '
 	test_when_finished "
 		git credential-cache exit --socket \"\$HOME/dir/socket\" &&
 		rmdir \"\$HOME/dir\"
@@ -78,7 +81,8 @@
 	test_path_is_socket "$HOME/dir/socket"
 '
 
-test_expect_success "use custom XDG_CACHE_HOME even if xdg socket exists" '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure "use custom XDG_CACHE_HOME even if xdg socket exists" '
 	test_when_finished "
 		git credential-cache exit &&
 		sane_unset XDG_CACHE_HOME
@@ -101,7 +105,8 @@
 	test_path_is_socket "$XDG_CACHE_HOME/git/credential/socket"
 '
 
-test_expect_success 'use user socket if user directory exists' '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure 'use user socket if user directory exists' '
 	test_when_finished "
 		git credential-cache exit &&
 		rmdir \"\$HOME/.git-credential-cache/\"
@@ -117,7 +122,8 @@
 	test_path_is_socket "$HOME/.git-credential-cache/socket"
 '
 
-test_expect_success SYMLINKS 'use user socket if user directory is a symlink to a directory' '
+# https://github.com/me-and/Cygwin-Git/issues/51
+test_expect_failure SYMLINKS 'use user socket if user directory is a symlink to a directory' '
 	test_when_finished "
 		git credential-cache exit &&
 		rmdir \"\$HOME/dir/\" &&

--- a/git-v2.51.0/t/t3070-wildmatch.sh	2022-12-11 00:32:48.000000000 +0000
+++ b/git-v2.51.0/t/t3070-wildmatch.sh	2022-12-11 15:08:33.604989200 +0000
@@ -119,7 +119,18 @@
 	then
 		if test -e .git/created_test_file
 		then
-			test_expect_success EXPENSIVE_ON_WINDOWS "$match_function (via ls-files): match '$pattern' '$text'" "
+			# https://github.com/me-and/Cygwin-Git/issues/42
+			# This is a very rough first attempt to characterise
+			# what will and will not cause faulures; it may be
+			# completely incorrect.
+			if [[ "$pattern" = *\\* ]] && [[ "$pattern" != *\\\\-* ]] && [[ "$pattern" != *\\\\,* ]] && [[ "$pattern" != *\[\\\[* ]] && ! [[ "$pattern" = '[\1-\3]' && "$text" = 3 ]]
+			then
+				local result=failure
+			else
+				local result=success
+			fi
+
+			test_expect_$result EXPENSIVE_ON_WINDOWS "ALPHA $match_function (via ls-files): match '$pattern' '$text'" "
 				printf '%s' '$text' >expect &&
 				git$ls_files_args ls-files -z -- '$pattern' >actual.raw 2>actual.err &&
 				$match_stdout_stderr_cmp
@@ -131,7 +142,17 @@
 	then
 		if test -e .git/created_test_file
 		then
-			test_expect_success EXPENSIVE_ON_WINDOWS "$match_function (via ls-files): no match '$pattern' '$text'" "
+			# https://github.com/me-and/Cygwin-Git/issues/42
+			# This is a very rough first attempt to characterise
+			# what will and will not cause faulures; it may be
+			# completely incorrect.
+			if [[ "$pattern" = *\\\\* ]]
+			then
+				local result=failure
+			else
+				local result=success
+			fi
+			test_expect_$result EXPENSIVE_ON_WINDOWS "$match_function (via ls-files): no match '$pattern' '$text'" "
 				>expect &&
 				git$ls_files_args ls-files -z -- '$pattern' >actual.raw 2>actual.err &&
 				$match_stdout_stderr_cmp

--- a/git-v2.51.0/t/t5580-unc-paths.sh
+++ b/git-v2.51.0/t/t5580-unc-paths.sh
@@ -40,28 +40,33 @@ test_expect_success setup '
 	test_commit initial
 '
 
-test_expect_success clone '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure clone '
 	git clone "file://$UNCPATH" clone
 '
 
-test_expect_success 'clone without file://' '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure 'clone without file://' '
 	git clone "$UNCPATH" clone-without-file
 '
 
-test_expect_success 'clone with backslashed path' '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure 'clone with backslashed path' '
 	BACKSLASHED="$(echo "$UNCPATH" | tr / \\\\)" &&
 	git clone "$BACKSLASHED" backslashed
 '
 
-test_expect_success fetch '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure fetch '
 	git init to-fetch &&
 	(
 		cd to-fetch &&
 		git fetch "$UNCPATH" main
 	)
 '
 
-test_expect_success push '
+# https://github.com/cygporter/git/issues/43
+test_expect_failure push '
 	(
 		cd clone &&
 		git checkout -b to-push &&
@@ -78,6 +83,7 @@ test_expect_success MINGW 'remote nick cannot contain backslashes' '
 '
 
-test_expect_success 'unc alternates' '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure 'unc alternates' '
 	tree="$(git rev-parse HEAD:)" &&
 	mkdir test-unc-alternate &&
 	(

