--- a/git-v2.51.0/t/t5580-unc-paths.sh
+++ b/git-v2.51.0/t/t5580-unc-paths.sh
@@ -40,28 +40,33 @@ test_expect_success setup '
 	test_commit initial
 '
 
-test_expect_success clone '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure clone '
 	git clone "file://$UNCPATH" clone
 '
 
-test_expect_success 'clone without file://' '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure 'clone without file://' '
 	git clone "$UNCPATH" clone-without-file
 '
 
-test_expect_success 'clone with backslashed path' '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure 'clone with backslashed path' '
 	BACKSLASHED="$(echo "$UNCPATH" | tr / \\\\)" &&
 	git clone "$BACKSLASHED" backslashed
 '
 
-test_expect_success fetch '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure fetch '
 	git init to-fetch &&
 	(
 		cd to-fetch &&
 		git fetch "$UNCPATH" main
 	)
 '
 
-test_expect_success push '
+# https://github.com/cygporter/git/issues/43
+test_expect_failure push '
 	(
 		cd clone &&
 		git checkout -b to-push &&
@@ -78,6 +83,7 @@ test_expect_success MINGW 'remote nick cannot contain backslashes' '
 '
 
-test_expect_success 'unc alternates' '
+# https://github.com/cygporter/git/issues/59
+test_expect_failure 'unc alternates' '
 	tree="$(git rev-parse HEAD:)" &&
 	mkdir test-unc-alternate &&
 	(
